src/test/java/uk/gov/hmcts/reform/authorisation/validators/ServiceAuthTokenValidatorTest.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.reform.authorisation.validators;
+
+import feign.FeignException;
+import feign.Response;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.reform.authorisation.ServiceAuthorisationApi;
+import uk.gov.hmcts.reform.authorisation.exceptions.InvalidTokenException;
+import uk.gov.hmcts.reform.authorisation.exceptions.ServiceException;
+
+import java.util.Collections;
+
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class ServiceAuthTokenValidatorTest {
+    private final ServiceAuthorisationApi api = mock(ServiceAuthorisationApi.class);
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    @Test
+    public void shouldValidateServiceAuthToken() {
```

Comment:
I found it unclear why the test calls `validate` twice and checks if two calls were made. Could we update the test name to reflect why?